Cargo.toml

```diff
@@ -8,10 +8,10 @@ members = [ "build/salva2d", "build/salva3d", "examples2d", "examples3d" ]
 [patch.crates-io]
 salva2d = { path = "./build/salva2d" }
 salva3d = { path = "./build/salva3d" }
-#rapier2d = { git = "https://github.com/dimforge/rapier" }
-#rapier3d = { git = "https://github.com/dimforge/rapier" }
-#rapier_testbed2d = { git = "https://github.com/dimforge/rapier" }
-#rapier_testbed3d = { git = "https://github.com/dimforge/rapier" }
+rapier2d = { git = "https://github.com/rezural/rapier", branch = "harness" }
+rapier3d = { git = "https://github.com/rezural/rapier", branch = "harness"  }
+rapier_testbed2d = { git = "https://github.com/rezural/rapier", branch = "harness"  }
+rapier_testbed3d = { git = "https://github.com/rezural/rapier", branch = "harness"  }
```

Comment:
Now that https://github.com/dimforge/rapier/pull/81 is merged, this can be switched back to the dimforge repos.

Reply:
OK, I have reverted that.